isis/src/control/apps/jigsaw/jigsaw.cpp

```diff
@@ -339,6 +347,53 @@ namespace Isis {
       throw IException(e, IException::User, msg, _FILEINFO_);
     }
 
+    if (ui.GetBoolean("OUTPUT_ADJUSTED_CSMSTATE")) {
+      
+      // Go thru each file in cubelist
+      SerialNumberList *snList = new SerialNumberList(cubeList);
+      for (int i = 0; i < snList->size(); i++) {        
+        QString filename = snList->fileName(i);
+        
+        // Generate ISD from cube
+        json props;
+        json isd = ale::load(filename.toStdString(), props.dump(), "ale", false, true, false);
```

Comment:
We should throw a try/catch around this load call. It will likely fail at some point as not all instruments have an `IsisLabelIsisSpice` driver